### DISCUSSION TITLE
Google Authenticator

### DISCUSSION THREAD

**Discussion by Shorider** [Security-Onion-Solutions/securityonion#7941] [Unsupported Versions]
Good morning,

I am currently running Security Onion 2.3.120.  I am trying to add an authenticator to my account and have followed the instructions of hitting the user icon in the upper right corner, going to settings, clicking on the Security tab.  I tried scanning the QR code and also tried inputting the long letter/number phrase.  Google authenticator pops up my 6 digit number code, I put it in Security Onion where it asks me to, click on Activate MFA and get the following:

Unable to save settings:  The provided authentication code is invalid, please try again.

I have tried speed running this in case it is all time sensitive from logging into Security Onion, scanning the QR code and putting in the 6 digit code as fast as possible, but I get the same result.  I have tried refreshing the page and immediately putting in the long code into the authenticator or scanning the QR code, putting in the 6 digit code the authenticator spits out and get the same error.  I am also deleting the account on Google Authenticator between each attempt.  

Am I missing something that I need to change in the user settings to allow this?

**Comment by dougburks** [comment]
Check the date and time on all devices.
